### PR TITLE
Sets emissivity index for human body

### PR DESCRIPTION
**Pull request by krishnadurai**
Implements write function for MLX thermopile
Implements CRC8 check for reading and writing

### CODE REVIEW

**Review by mmaurya50 — COMMENTED** [review]
#define MLX90614_EMISS 0x24 must be #define MLX90614_EMISS 0x04 as per 5.1 of[ [pdf]
(https://www.melexis.com/en/documents/documentation/application-notes/application-note-mlx90614-changing-emissivity-setting)

File: TempSensor.ino [hunk 1]
```diff
@@ -124,8 +181,20 @@ float MLX90614::readTemp(uint8_t reg) {
   return temp;
 }
 
-/*********************************************************************/
+void MLX90614::setEmissivity(float emissivity) {
+  uint16_t rawOld = read16(MLX90614_EMISS);
+  uint16_t rawNew = round(emissivity * 65535);
```
Comment (mmaurya50) [inline TempSensor.ino:187]: uint16_t rawNew = round(emissivity * 65535-1);

**Comment by krishnadurai** [comment]
> #define MLX90614_EMISS 0x24 must be #define MLX90614_EMISS 0x04 as per 5.1 of[ [pdf]
> (https://www.melexis.com/en/documents/documentation/application-notes/application-note-mlx90614-changing-emissivity-setting)

Sure I'll make this change

**Comment by krishnadurai** [comment]
This is ready for review and merge.

**Review by yashtambi — APPROVED** [review]

**Review by mmaurya50 — APPROVED** [review]

File: TempSensor.ino (hunk 1 above)
Comment (krishnadurai) [reply TempSensor.ino:187]: I'll address this change

**Pull request merged**